dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/note/hibernate/Note.hbm.xml

```diff
@@ -11,7 +11,13 @@
     <id name="id" column="noteid">
       <generator class="native" />
     </id>
-    &identifiableProperties;
+
```

Comment:
why do we need this mapping if we use JDBC?

Reply:
We still need it because tracker importer is adding notes that are defined in the enrollments/events and we are persisting using hibernate there.